moonbeam-types-bundle/index.ts

```diff
@@ -47,11 +47,17 @@ export const moonbeamDefinitions = {
         Candidate: {
           validator: "AccountId",
           fee: "Perbill",
-          nominators: "OrderedSet<Bond<AccountId, Balance>>",
+          nominators: "Vec<Bond>",
           total: "Balance",
           state: "ValidatorStatus<RoundIndex>",
         },
-        OrderedSet: "Vec",
+        Bond: {
+          owner: "AccountId",
+          amount: "Balance",
+        },
+        ValidatorStatus: {
+          _enum: ["Active", "Idle", "Leaving(BlockNumber)"],
```

Comment:
```suggestion
          _enum: ["Active", "Idle", "Leaving(RoundIndex)"],
```

I know it says BlockNumber in the ValidatorStatus definition, but that is actually generic. The instantiated type only RoundIndex and I actually removed the generic type parameter for that reason in #189 but for now, it is `RoundIndex`

Reply:
To be fair, won't make a difference because RoundIndex = BlockNumber = u32